MotMystere.cpp

```diff
@@ -2,19 +2,52 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+
+using namespace std;
 
 int main()
 {
-    std::cout << "Hello World!\n";
+   
+    /*string motMystere("Procrastination");
+    //1 : On demande de saisir un mot
+    cout << "Entrer un mot : ";
+    std::cin >> motMystere;
+    cout << "Le mot est :" << motMystere;
+    return 0;*/
+
+    //3 : On demande à l'utilisateur quel est le mot mystère
```

Comment:
Could you remove the comments? In this case, comments refer to your program specification, and previous tests. It makes the code harder to understand visually, and prevents other people from understanding what exactly is going on.
For example, I could ask myself: 
- Are the comments necessary?
- Are they a previously implemented but now useless feature, or are they destined to be used in the future?
- Do they refer to something I don't understand from this code?

And so on.

So it is good practice to remove these kind of comments once you're done implementing a feature

Reply:
done